docs/DeveloperGuide.md

```diff
@@ -966,7 +996,7 @@ The `organise` command will set a meet up with the time and place for all partic
 :information_soruce: **Command Formats:** <br>
 
 * `organise INDEX` : Takes the recommendation at the specified `INDEX` and saves it.
-* `organise INDEX_1 INDEX_2 ... d/DAY T/START_TIME END_TIME l/LOCATION` : Creates a new meet up with the specified participants and information.
+* `organise INDEX_1 INDEX_2 … d/DAY T/START_TIME END_TIME l/LOCATION` : Creates a new meet up with the specified participants and information.
```

Comment:
Thought we agreed to change this to `...`.

Reply:
That was User Guide la lol.